### PR TITLE
PCRM-3942 Create Mesh Stack Workflow

### PR DESCRIPTION
**Pull request by joeds13**

### CODE REVIEW

File: terraform/stacks/mesh/modules.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module "mesh" {
+  source = "git::https://github.com/joeds13/spine-core-aws-common.git//terraform/mesh_lambdas?ref=js-mesh-lambdas"
```
Comment (josh-scargill-nhsd) [inline terraform/stacks/mesh/modules.tf:2]: Should be stored somewhere better long term?

**Review by josh-scargill-nhsd — APPROVED** [review]

**Comment by josh-scargill-nhsd** [comment]
I assume this has been tested and works? :D

File: terraform/stacks/mesh/modules.tf (hunk 1 above)
Comment (joeds13) [reply terraform/stacks/mesh/modules.tf:2]: Yeah, that's just my fork it'll get merged upstream to `nhsdigital/spine-core-aws-common`

Depending on how Ramen progresses and whether we want the Mesh stack in sooner, we can choose if we want to merge this or wait until it's merged upstream

**Comment by joeds13** [comment]
> I assume this has been tested and works? :D

I've sent and received messages from Integration Mesh using this stack, so simple functionality yes!

Ramen is working with Solutions Assurance to understand exactly how we scope testing and approval of this as it does not sit "inside" the List Rec scope

**Review by abbas-khan8 — APPROVED** [review]

**Comment by abbas-khan8** [comment]
Assuming it works and the failing tests are inevitably going to be fixed it looks good to me

**Review by jamesnunnnhs — APPROVED** [review]

**Review by abbas-khan8 — APPROVED** [review]

**Pull request merged**